apps/src/bounce/bounce.js

```diff
@@ -925,9 +925,22 @@ Bounce.reset = function(first) {
   // Soft buttons
   var softButtonCount = 0;
   for (i = 0; i < Bounce.softButtons_.length; i++) {
-    document.getElementById(Bounce.softButtons_[i]).style.display = 'inline';
+    document.getElementById(Bounce.softButtons_[i]).classList.add('visible');
     softButtonCount++;
   }
+  // We need to apply a unique margin rule to the last visible button displayed.
+  // softButtons_ can include any of ['leftButton', 'rightButton', 'downButton', 'upButton']
+  // The #downButton element is always displayed last, despite the above order.
+  // If the level uses a 'downButton', this will be the last button.
+  // If not, it will be the last item in the array.
```

Comment:
ahhhh, i missed this when we were pairing 🤦🏻‍♀️ having to deal with this added complexity and tight coupling makes this fix less beneficial -- it kind of ends up adding more tech debt. i think we should either explore a more react-focused approach and deal with this margin bug (which is pretty minor) in the meantime